src/style.css

```diff
@@ -0,0 +1,124 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+* {
+  box-sizing: border-box;
+  font-family: Avenir, Helvetica, sans-serif;
```

Comment:
set it to the `html` tag selector and let inputs inherit it

Reply:
not fixed, move this to `html` selector